cpp/log/etcd_consistent_store_test.cc

```diff
@@ -736,7 +735,7 @@ TEST_F(EtcdConsistentStoreTest, TestCleansUpToNewSTH) {
   {
     const StatusOr<int64_t> num_cleaned(CleanupOldEntries());
     ASSERT_OK(num_cleaned.status());
-    EXPECT_EQ(2, num_cleaned.ValueOrDie());
+    EXPECT_EQ(5, num_cleaned.ValueOrDie());
```

Comment:
The faster cleaner doesn't do a `Get` first, and `EtcdDeleteKeys` doesn't report the number of keys deleted, indeed. It'll now repeatedly try to delete keys that are still in the mapping, and it's the signer that will be cleaning the stale mapping entries.